man/ukify.xml

```diff
@@ -112,6 +112,22 @@
 
       <para>The output files must not exist.</para>
     </refsect2>
+
+    <refsect2>
+      <title><command>inspect</command></title>
+
+      <para>Display information about the given PE, listing all known sections and
+      print the content of each (if text readable) together with displaying its size and
+      <literal>sha256</literal>.
+      This mode has already a list of default sections to show, but it is possible to specify
+      additional custom fields via the <option>--section=</option> option.
+      Each field is and must be defined as <literal>binary</literal> or <literal>text</literal>, so that the program will know whether to read the section content or not.
```

Comment:
Strike "so that the program will know whether to read the section content or not.".